src/runtime/contrib/dnnl/dnnl_json_runtime.cc

```diff
@@ -191,21 +228,30 @@ class DNNLJSONRuntime : public JSONRuntimeBase {
     dnnl::memory::dims padding_dims_r = {PH_R, PW_R};
 
     // Memory descriptions.
-    auto conv_src_md = dnnl::memory::desc(src_dims, dt::f32, tag::any);
-    auto conv_weights_md = dnnl::memory::desc(weights_dims, dt::f32, tag::any);
-    auto conv_bias_md = dnnl::memory::desc(bias_dims, dt::f32, tag::any);
-    auto conv_dst_md = dnnl::memory::desc(dst_dims, dt::f32, tag::nchw);
+    auto conv_src_md = dnnl::memory::desc(src_dims, dt::f32, src_df);
+    auto conv_weights_md = dnnl::memory::desc(weights_dims, dt::f32, weight_df);
+    auto conv_bias_md = dnnl::memory::desc(bias_dims, dt::f32, tag::x);
+    auto conv_dst_md = dnnl::memory::desc(dst_dims, dt::f32, dst_df);
 
     // Covn2d description.
     auto conv_desc = dnnl::convolution_forward::desc(
```

Comment:
Check if there's `bias` and then decide which function to call, instead of calling the non-bias version and recreate a bias version to override?

Reply:
fixed.